packages/input-text/src/TextInput.js

```diff
@@ -5,15 +5,37 @@ import disabledTooltip, {
   disabledTooltipProps
 } from "@crave/farmblocks-hoc-disabled-tooltip";
 import formInput, { formInputProps } from "@crave/farmblocks-hoc-input";
+import Link from "@crave/farmblocks-link";
 
 import Container from "./styledComponents/TextInput";
 const EnhancedInput = compose(disabledTooltip, formInput)("input");
 
-const TextInput = props => (
-  <Container>
-    <EnhancedInput {...props} displayBlock />
-  </Container>
-);
+const searchProps = {
+  icon: "wg-search",
+  clearable: true
```

Comment:
do we need clearable? dont all type=search inputs have this feature by default? how is it on our specs?

Reply:
allsearch fields have the clear button, see slack discussion https://cravefoodservices.slack.com/archives/C1AB7U58W/p1516300464000496

Reply:
I though on having `clearable` so it could be used by other types of inputs.
But I we can do this only when/if we need.